pwiz_tools/Skyline/CommandArgs.cs

```diff
@@ -1215,6 +1237,37 @@ public bool ImportingSearch
             (c, p) => c.PredictCoVName = p.Value) { WrapValue = true };
         public static readonly Argument ARG_TRAN_PREDICT_OPTDB = new DocArgument(@"tran-predict-optdb", () => GetDisplayNames(Settings.Default.OptimizationLibraryList),
             (c, p) => c.PredictOpimizationLibraryName = p.Value) { WrapValue = true };
+
+        public static readonly Argument ARG_FULL_SCAN_PRECURSOR_ISOTOPES = new DocArgument(@"full-scan-precursor-isotopes",
+                () => Enum.GetNames(typeof(FullScanPrecursorIsotopes)),
+                (c, p) => c.FullScanPrecursorIsotopes = (FullScanPrecursorIsotopes) Enum.Parse(typeof(FullScanPrecursorIsotopes), p.Value))
+            { WrapValue = true };
+        public static readonly Argument ARG_FULL_SCAN_PRECURSOR_ANALYZER = DocArgument.FromEnumType<FullScanMassAnalyzerType>(@"full-scan-precursor-analyzer",
+            (c, p) => c.FullScanPrecursorMassAnalyzerType = p);
+        public static readonly Argument ARG_FULL_SCAN_PRECURSOR_THRESHOLD = new DocArgument(@"full-scan-precursor-threshold", NUM_VALUE,
+                (c, p) => c.FullScanPrecursorThreshold = p.GetValueDouble(0, 100)) { WrapValue = true };
+        public static readonly Argument ARG_FULL_SCAN_PRECURSOR_ISOTOPE_ENRICHMENT = new DocArgument(@"full-scan-precursor-isotope-enrichment",
+                () => GetDisplayNames(Settings.Default.IsotopeEnrichmentsList),
+                (c, p) => c.FullScanPrecursorIsotopeEnrichment = p.Value)
+            { WrapValue = true };
+        public static readonly Argument ARG_FULL_SCAN_PRECURSOR_IGNORE_SIM = new DocArgument(@"full-scan-precursor-ignore-sim",
+                (c, p) => c.FullScanPrecursorIgnoreSimScans = p.IsNameOnly || bool.Parse(p.Value))
+            { WrapValue = true, OptionalValue = true };
+
+        public static readonly Argument ARG_FULL_SCAN_ACQUISITION_METHOD = new DocArgument(@"full-scan-acquisition-method",
+                () => FullScanAcquisitionMethod.AVAILABLE.Select(m => m.Name).ToArray(),
+                (c, p) => c.FullScanAcquisitionMethod = FullScanAcquisitionMethod.FromName(p.Value))
+            { WrapValue = true };
+        public static readonly Argument ARG_FULL_SCAN_PRODUCT_ANALYZER = DocArgument.FromEnumType<FullScanMassAnalyzerType>(@"full-scan-product-analyzer",
+            (c, p) => c.FullScanProductMassAnalyzerType = p);
+        public static readonly Argument ARG_FULL_SCAN_PRODUCT_ISOLATION_SCHEME = new DocArgument(
+                @"full-scan-isolation-scheme",
+                () => string.Join(TextUtil.SEPARATOR_CSV + @" ",
+                    GetDisplayNames(Settings.Default.IsolationSchemeList)
+                        .Append("or a result filepath to import the scheme from")),
```

Comment:
I added some magic handling here where the same argument can be either a result filepath to import the isolation scheme from or the name of an existing isolation scheme. Sound good @brendanx67 ?